app/services/feeds/import.rb

```diff
@@ -141,6 +141,7 @@ def create_articles_from_user_feed(user, feed)
           organization_id: nil,
         )
 
+        subscribe_author_to_comments(user, article)
```

Comment:
well that sure sounds succinct :)

Do we need to pass user here, or does the article contain enough information to create the subscription automatically? Since user was loaded to validate the relation during create (we load user by id) it looks like the associated model is already present in the cache - `article.user` returns an object without hitting the db a second time.

Reply:
The query for user/the User object is passed in from another method so I don't think there would be a second query.